### PR TITLE
feat: CNTK Export Provider

### LINKED ISSUE #754: V2 - No CNTK Export
Hi Noticed the new V2 does not have the CNTK export. Is this something that is in the pipeline?

### PR DESCRIPTION
**Pull request by wbreza**
Adds CNTK export provider into v2

Resolves #754

### CODE REVIEW

File: src/common/localization/es-cl.ts [hunk 1]
```diff
@@ -289,6 +289,10 @@ export const spanish: IAppStrings = {
                             tagged: "Solo activos etiquetados",
                         },
                     },
+                    testTrainSplit: {
+                        title: "Prueba/tren Split",
```
Comment (tbarlow12) [inline src/common/localization/es-cl.ts]: La división para entrenar y comprobar

File: src/common/localization/es-cl.ts [hunk 2]
```diff
@@ -289,6 +289,10 @@ export const spanish: IAppStrings = {
...
+                        description: "La división del tren de prueba que se utilizará para los datos exportados",
```
Comment (tbarlow12) [inline src/common/localization/es-cl.ts]: La división de datos para utilizar entre el entrenamiento y la verificación

**Comment by pjlittle** [comment]
Yeah looks good - like Jacopo, I was wondering if anyone had tried using the output w/ CNTK or on the DSVM.

**Comment by wbreza** [comment]
@JacopoMangiavacchi @pjlittle - I didn't load up the data into any ML pipeline / tool, but compared the output from a project in v1 vs v2 and is the same.  I'll complete the merge for now but will also attempt to load training data into some model.

**Pull request merged**